l10n_ro_account_edi_ubl/models/account_edi_format.py

```diff
@@ -41,6 +41,26 @@ def _export_cius_ro(self, invoice):
     def _export_invoice_filename(self, invoice):
         return f"{invoice.name.replace('/', '_')}_cius_ro.xml"
 
+    def _find_value(self, xpath, xml_element, namespaces=None):
+        res = None
+        try:
```

Comment:
In ce conditii se intampla asta, putem face un test

Reply:
in conditiile in care factura preluata de la ANAF nu are toate namespace-urile necesare. Ex: Factura primita de la Dedeman, nu poate fi citita, cand lxml cauta namespace in fisier gaseste doar unu. 
Problema nu este la "Dedeman", ci la lxml ca da eroare daca ceea ce gaseste nu se afla in namespace.